iOS/MMIWG2S/Utilities/MmiwUtility.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MmiwUtility.swift
+//  MMIWG2S
+//
+//  Created by Stephen Emery on 7/12/21.
+//  Copyright © 2021 Google LLC. All rights reserved.
+//
+
+import UIKit
+import ARKit
+
+class MmiwUtility {
+    static var faceViewController: UIViewController {
+        if ARFaceTrackingConfiguration.isSupported {
+            return FacesARViewController()
+        } else {
+            return FacesViewController()
+        }
+    }
+
+    // MARK: UserDefault Helper methods
+    enum UserDefaultKeys: String {
```

Comment:
See the [Swift Language Guide on Enumerations](https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html)

> Give enumeration types singular rather than plural names, so that they read as self-evident